types/popper.js/index.d.ts

```diff
@@ -1,58 +1,74 @@
-// Type definitions for popper.js v0.4.0
+// Copyright 2017 Canva Inc. All Rights Reserved.
+
+// Type definitions for popper.js v1.8.5
 // Project: https://github.com/FezVrasta/popper.js/
-// Definitions by: rhysd <https://rhysd.github.io>
+// Definitions by: rhysd <https://rhysd.github.io>, joscha <https://github.com/joscha>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
 declare namespace Popper {
-	export interface PopperOptions {
-		placement?: string;
-		gpuAcceleration?: boolean;
-		offset?: number;
-		boundariesElement?: string | Element;
-		boundariesPadding?: number;
-		preventOverflowOrder?: ("left" | "right" | "top" | "bottom")[];
-		flipBehavior?: string | string[];
-		modifiers?: string[];
-		modifiersIgnored?: string[];
-		removeOnDestroy?: boolean;
-		arrowElement?: string | Element;
-	}
-	export class Modifiers {
-		applyStyle(data: Object): Object;
-		shift(data: Object): Object;
-		preventOverflow(data: Object): Object;
-		keepTogether(data: Object): Object;
-		flip(data: Object): Object;
-		offset(data: Object): Object;
-		arrow(data: Object): Object;
-	}
-	export interface Data {
-		placement: string;
-		offsets: {
-			popper: {
-				position: string;
-				top: number;
-				left: number;
-			};
-		};
-	}
+  export interface PopperOptions {
+    placement?: string;
+    boundariesPadding?: number;
+    modifiers?: {
+      order?: number,
+      enabled?: boolean,
+      'function'?: Function,
```

Comment:
Note that this is deprecated, now it's just `fn`. The old one is still supported tho